### DISCUSSION TITLE
Cisco IOS devices and multi line banners with Ascii char 3 creates unnecessary diff

### DISCUSSION THREAD

**Discussion by falz** [napalm-automation/napalm#1771] [General]
We're pushing configs to Cisco IOS-XE devices (ASR920, 16.9.x), and am working on moving from MERGE to FULL.  We're aware of an issue with the `^C` character actually having to be `chr(3)` per https://napalm.readthedocs.io/en/latest/support/ios.html#banner

Before pushing, we scrub config for this using this, which I believe to be correct, and it does properly commit:

```python
def sanitize_config(dirty_config_str):
        clean_config_str = re.sub(r'\^C', '\x03', dirty_config_str)
        return(clean_config_str)
```

However, no matter what, we get a diff showing the banners added and being removed, weather we use `load_replace_candidate()` or `load_merge_candidate()`

Our goal is to have NO diff to signal to the operator (yes, a human still) that things are good. 

Curious if others have this issue?

**Comment by ktbyers** [comment]
@falz Let me test if I can reproduce your behavior or not.

**Comment by falz** [comment]
@ktbyers curious if you were able to investigate this? if no issue curious details of what system you'ren testing on (IOS type/version/etc)

cheers
